### ISSUE TITLE
trailing_closure rule is not triggered for contains(where:)

### ISSUE THREAD

**Issue by Igor-Palaguta** [realm/SwiftLint#2587]
### Describe the bug

trailing_closure rule is not triggered for contains(where:) function

```
let containsHidden = views.contains(where: { $0.isHidden })
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
0.30.1

* Installation method used (Homebrew, CocoaPods, building from source, etc)?
package file from github

* Paste your configuration file:
```yml
opt_in_rules:
  - trailing_closure
```

* Which Xcode version are you using (check `xcode-select -p`)?
Version 10.0 (10A255)

* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`

```shell
echo "array.contains(where: { $0.isHidden })" | swiftlint lint --no-cache --use-stdin --enable-all-rules
```

This should be triggered, but not
```swift
let containsHidden = views.contains(where: { $0.isHidden })
```

**Comment by marcelofabri** [comment]
This works for me:

```
$ "array.contains(where: { $0.isHidden })" | swiftlint lint --no-cache --use-stdin --enable-all-rules

Loading configuration from '.swiftlint.yml'
<nopath>:1:1: warning: Trailing Closure Violation: Trailing closure syntax should be used whenever possible. (trailing_closure)
Done linting! Found 1 violation, 0 serious in 1 file.
$ swiftlint version
0.30.1
```

**Comment by Igor-Palaguta** [comment]
Just trailing new line is triggered

<img width="844" alt="screen shot 2019-01-24 at 17 38 14" src="https://user-images.githubusercontent.com/800044/51689219-e8d5f100-1ffe-11e9-9fc3-35f94a725b53.png">

**Comment by marcelofabri** [comment]
@Igor-Palaguta you're not running `echo`, so `swiftlint` is not getting the text (you probably just copied from my comment which I forgot to add `echo` even though I run it)

**Comment by Igor-Palaguta** [comment]
I think I found why it does not work
We added this to yml file:

```
trailing_closure:
  only_single_muted_parameter: true
```

**Comment by marcelofabri** [comment]
Closing this then, because `only_single_muted_parameter` was added to only trigger on calls where the closure is the only parameter and is muted, e.g. `array.map({ $0.isHidden})`

**Comment by Igor-Palaguta** [comment]
@marcelofabri Thanks for clarification. I did not catch what muted means. So will wait for `only_single_parameter`
